### PR TITLE
Setup code formatting and improve project structure

### PR DESCRIPTION
**Pull request by BeniRupp**
I've added Black for automated code formatting and restructured the files.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Wie du mitmachen kannst
+
+## Code Formatierung
+Wir nutzen [Black](https://black.readthedocs.io/en/stable/#) zum automatisierten Formatieren von Python Code.
+Black kannst du auch [für deine Entwicklungsumgebung einrichten](https://black.readthedocs.io/en/stable/editor_integration.html).
```
Comment (RSchlenker) [inline CONTRIBUTING.md:39]: Missing usage documentation here
Please add: `How to install requirements txt?` and `how to run black for all files, maybe a git hook if possible?`

Comment (RSchlenker) [reply CONTRIBUTING.md:39]: Maybe also a short note on `venv` ?

File: README.md [hunk 2]
```diff
@@ -19,6 +19,6 @@ Folgende Vorkenntnisse sind idealerweise vorhanden, wenn dieses Spiel erweitert
 Gestartet wird das Spiel über die Konsole wie folgt:
 
 ```
-python3 game.py
+./play.py
 ```
```
Comment (RSchlenker) [inline README.md:25]: We should reference the CONTRIBUTING.md file here

**Review by RSchlenker — CHANGES_REQUESTED** [review]

File: CONTRIBUTING.md (hunk 1 above)
Comment (BeniRupp) [reply CONTRIBUTING.md:39]: I think we should switch to [pipenv](https://pipenv.readthedocs.io) to make things easier.

Comment (BeniRupp) [reply CONTRIBUTING.md:39]: A git hook sounds great! 😍

**Comment by BeniRupp** [comment]
@RSchlenker I have replaced all the venv stuff with a pipenv setup. Please have a look at 
75b0209. Can you install dependencies and run the game using the provided documentation?

File: Pipfile [hunk 3]
```diff
@@ -0,0 +1,12 @@
+[[source]]
+name = "pypi"
+url = "https://pypi.org/simple"
+verify_ssl = true
+
+[dev-packages]
+
+[packages]
+black = "*"
```
Comment (RSchlenker) [inline Pipfile]: ist das nicht eher ein dev-package :)

**Review by RSchlenker — APPROVED** [review]

**Review by RSchlenker — APPROVED** [review]

**Review by RSchlenker — APPROVED** [review]
Wonderful!

**Pull request merged**